lib/servers.php

```diff
@@ -273,4 +273,27 @@ private static function match($server, $user, $name) {
         }
         return false;
     }
+
+    private static function appendPpasswordAndUsername(array $server) {
```

Comment:
Is the double `p` a typo?

Reply:
More importantly, where does the need to have the properties "pass, user,  password, username" on the server come from? Why can't the `pass` and `user` be solely used?